components/editor/vim/vim.p5m

```diff
@@ -148,1694 +148,1713 @@ link path=usr/share/man/ru.UTF-8/man1/rvim.1 target=vim.1
 file path=usr/share/man/ru.UTF-8/man1/vimdiff.1
 file path=usr/share/man/ru.UTF-8/man1/vimtutor.1
 file path=usr/share/man/ru.UTF-8/man1/xxd.1
-file path=usr/share/vim/vim81/autoload/README.txt
-file path=usr/share/vim/vim81/autoload/RstFold.vim
-file path=usr/share/vim/vim81/autoload/ada.vim
-file path=usr/share/vim/vim81/autoload/adacomplete.vim
-file path=usr/share/vim/vim81/autoload/ccomplete.vim
-file path=usr/share/vim/vim81/autoload/clojurecomplete.vim
-file path=usr/share/vim/vim81/autoload/context.vim
-file path=usr/share/vim/vim81/autoload/contextcomplete.vim
-file path=usr/share/vim/vim81/autoload/csscomplete.vim
-file path=usr/share/vim/vim81/autoload/decada.vim
-file path=usr/share/vim/vim81/autoload/dist/ft.vim
-file path=usr/share/vim/vim81/autoload/getscript.vim
-file path=usr/share/vim/vim81/autoload/gnat.vim
-file path=usr/share/vim/vim81/autoload/gzip.vim
-file path=usr/share/vim/vim81/autoload/haskellcomplete.vim
```

Comment:
Should we use some var for 81/82/etc?

Reply:
I'd rather not to. Having variable in vim.p5m makes it a mess for comparison with `vimdiff` when new new stuff gets added to sample-manifest, because the variable is not in the sample-manifest. (I understand that it's a messy in version changes like this one, though still far easier for me to do the update right.)